ctapipe/coordinates/tests/test_ground_frame.py

```diff
@@ -0,0 +1,54 @@
+import astropy.units as u
+import pytest
+from astropy.coordinates import EarthLocation
+
+# LST-1, MAGIC-1, MAGIC-2
+locations = [
```

Comment:
If these are somewhat "official", we might include these somewhere else?  E.g. `ctapipe/utils` or whatever.  Might be beneficial for visualizations.

Reply:
These are used for tests, one should rely on the positions in the `subarray`.